switchdev_utils/config-service/README.md

```diff
@@ -33,11 +33,13 @@ List of SRIOV PF objects which need to be configured, and has the following attr
     - numOfVfs: Number of vfs to be configured on the sriov pf.  
     - switchdev: Boolean, Enable switchdev mode.  
     - vfTrust: Boolean, Configure vf trust.  
+    - mtu: Number, configure mtu for sriov_pf (it will be ignored if sriov_pf is a bond slave)
 - linuxBonds:  
 list of linux bond objects which need to be created and configured  
     - name: The name of the bond interface  
     - slaves: List of bond slaves interfaces (a subset of sriov pfs)  
     - bondingOptions: Bonding options of the bond interface  
+    - mtu: Number, configure mtu for bond interface and its slaves
```

Comment:
can you add a Note at the end of this section explaining the mtu behaviour when set on bond / pfs as describe in commit message ?

Reply:
done